packages/core/.storybook/manager-head.html

```diff
@@ -18,6 +13,10 @@
   body[localhost] .sidebar-item[id*='experimental'] {
     display: flex;
   }
+
+  body[localhost] [data-title='Internal APIs'] {
+    display: block;
+  }
```

Comment:
Added new section that is visible only during local development so we can see, test and document available internal components.